server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/java/JdkResolver.kt

```diff
@@ -59,14 +59,9 @@ class JdkResolver(
           null
     val javaHome = javaHomeFile?.let { bazelPathsResolver.resolveUri(it) }
 
-    val version =
-        if (targetInfo.hasJavaToolchainInfo())
-          targetInfo.javaToolchainInfo.sourceVersion
-        else
-          null
-
+    val version = null
```

Comment:
now when the version is null the mechanizm in `JdkCandidate` for the `version` field should be updated (in general we can remove `version` from `JdkCandidateData`)